core/domain/exp_jobs_one_off.py

```diff
@@ -693,3 +693,44 @@ def reduce(status, values):
             yield (status, len(values))
         else:
             yield (status, values)
+
+
+class CustomizationArgsValidationJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for validating all the customizations arguments of
+    Rich Text Components.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.ExplorationModel]
+
+    @staticmethod
+    def map(item):
+        if item.deleted:
+            return
+        err_dict = {}
+
+        try:
+            exploration = exp_services.get_exploration_from_model(item)
+        except Exception as e:
+            yield('Error %s in exploration' % str(e), [item.id])
```

Comment:
in exploration --> when loading exploration

Reply:
Done.